docs/development/custom-ca-citadel/README.md

```diff
@@ -0,0 +1,103 @@
+# Using a custom intermediate CA with Citadel
+
+## Background
+
+Imagine you work for a big company that runs its own PKI. Most of these companies own its own Root CA and generate
+intermediate CAs to sign Clients and/or Server Certificates.
+
+As Istio is capable of issuing certificates for every workload inside a mesh, it is possible to provide it a custom CA
+to sign the workload's certificates instead of using the self-signed one.
+
+
+## Requirements
+
+Ensure to have the following files:
+
+- `certs/ca-cert.pem`: Certificate of the Intermediate CA.
+- `certs/ca-key.pem`: Private key of the Intermediate CA.
+- `certs/root-cert.pem`: Certificate of the Root CA.
+- `certs/cert-chain.pem`: Certificate Chain. `cat certs/ca-cert.pem certs/root-cert.pem > certs/cert-chain.pem`.
+
+Also, it's required to know what `katalog/istio/sidecar-injection/configuration` was used some tunning will be needed.
```

Comment:
Typo: "tunning"

I would rephrase in "it's required to know what `katalog/istio/sidecar-injection/configuration` was used as some tunning will be needed" (add "as").